web-frontend/src/validation/nickname.tsx

```diff
@@ -8,5 +8,5 @@ export function isAlreadyExists(
 }
 
 export function isNicknameValid(nickname: string): boolean {
-  return /^[a-z0-9_-]+$/.test(nickname);
+  return /^[a-zA-Z0-9_-]+$/.test(nickname);
```

Comment:
What kind of validation this regex is doing!? Could you please add a description above!? You know regex is hard to follow! 😆